docs/en/modules.md

```diff
@@ -170,6 +170,25 @@ modules: {
 }
 ```
 
+#### Register Global Action in Namespaced Modules
+
+If you want to register global actions in namespaced modules, you can mark it with `root: true`. For example:
+
+``` js
+modules: {
+  foo: {
+    namespaced: true,
+
+    actions: {
+      someAction: {
+        root: true,
+        handler: () => {}
```

Comment:
IMO, it would be better we explicitly state `handler` is actual action definition and also has some code example for it so that the readers easily understand how they write their `handler` at a glance.